modules/space/rendering/renderableorbitalkepler.cpp

```diff
@@ -158,7 +155,7 @@ namespace {
         "StartRenderIdx",
         "Contiguous Starting Index of Render",
         "Index of object in renderable group to start rendering (all prior objects will "
-        "be ignored).",
+        "be ignored).", // @TODO
```

Comment:
I don't think I was sure what this menas.  Get back to it

Reply:
Basically, if a RenderableOrbitalKepler has 50000 objects and `StartRenderIdx` is 40000, it only renders objects 40001-50000.
But agree that it could be formulated better